src/main/java/com/example/bookstore/model/Book.java

```diff
@@ -0,0 +1,29 @@
+package com.example.bookstore.model;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+import java.math.BigDecimal;
+import lombok.Data;
+
+@Data
```

Comment:
@Data is a combination of multiple annotations, some of them (equals and hashcode, toString) can cause troubles with JPA entities, so please use only what you need - @ Getter/Setter